org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/variables/VariableUtils.java

```diff
@@ -142,10 +143,20 @@ public static boolean hasChildren(Value value, boolean includeStatic) {
             }
         } catch (AbsentInformationException ex) {
             int argId = 0;
-            for (Value argValue : stackFrame.getArgumentValues()) {
-                Variable var = new Variable("arg" + argId, argValue);
-                var.argumentIndex = argId++;
-                res.add(var);
+            try {
```

Comment:
it is a jdi bug which throws InteralException on getArgumentValues:

https://community.oracle.com/thread/1523023